python/hail/expr/types.py

```diff
@@ -269,6 +320,8 @@ def __str__(self):
     def _eq(self, other):
         return isinstance(other, TFloat64)
 
+    def _convert_to_json(self, x):
+        return float(x)
```

Comment:
By typecheck, it already has to be float, doesn't it?

Reply:
meant to be convert_from_json. JSON returns "NaN" strings

Reply:
it'll fail the tutorial without this, I think.

Reply:
yep, failed exactly where I expected. I fixed without retesting locally.